deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -81,39 +85,106 @@ spec:
-    
+
     3.3. For iSCSI, perform these three steps.
-    
+
     3.3.1. Make sure that the login used to log in to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$> service iscsid restart`.
-    
-    3.3.2. Discover and log into at least two iSCSI targets on the relevant storage
-    systems.
-    
+
+    3.3.2. Discover and log into at least two iSCSI targets on the relevant storage systems. (NOTE: Without at least two ports, multipath device will not be created.)
+
     ```bash
-    $ iscsiadm -m discoverydb -t st -p ${storage system iSCSI port IP}:3260
-    --discover
-    $ iscsiadm -m node -p ${storage system iSCSI port IP/hostname} --login
+    $ iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1}:3260 --discover
+    $ iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1} --login
+   
+    $ iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2}:3260 --discover
+    $ iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2} --login
     ```
-    
+
     3.3.3. Verify that the login was successful and display all targets that you logged into. The portal value must be the iSCSI target IP address.
-    
+
     ```bash
     $ iscsiadm -m session --rescan
     Rescanning session [sid: 1, target: {storage system IQN},
-    portal: {storage system iSCSI port IP},{port number}
+    portal: {STORAGE-SYSTEM-iSCSI-PORT-IP1},{port number}
+    portal: {STORAGE-SYSTEM-iSCSI-PORT-IP2},{port number}
+    ```
+
+
+
+    ## Configuring k8s secret and storage class
+    In order to use the driver, create the relevant storage classes and secrets, as needed. 
+    Note: This section can be done also after the operator installation.
+
+    This section describes how to:
+     1. Create a storage system secret - to define the storage system credentials (user and password) and its address.
+     2. Configure the k8s storage class - to define the storage system pool name, secret reference, SpaceEfficiency (thin, compressed, or deduplicated) and fstype(xfs\ext4).
+
+    #### 1. Create an array secret 
+    Create a secret file as follows and update the relevant credentials:
+
+    ```
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name: <VALUE-1>
+      namespace: kube-system
+    type: Opaque
+    stringData:
+      management_address: <VALUE-2,VALUE-3> # Array management addresses
+      username: <VALUE-4>                   # Array username  
+    data:
+      password: <VALUE-5 base64>            # Array password
+    ```
+
+    Apply the secret:
+    
+    ```bash
+    $ kubectl apply -f array-secret.yaml
     ```
 
+    #### 2. Create storage classes
+
+    Create a storage class yaml file `storageclass-gold.yaml` as follows, with the relevant capabilities, pool and, array secret:
```

Comment:
Create a storage class as follows, because in the secret section, you don't specify the file name.

Reply:
added the file name to the secret as well.

Reply:
"yaml file" or "file", please be consistent.

Reply:
what about:

Create a 'storageclass-gold.yaml' file as follows, with the relevant capabilities, pool, and array secret: